src/main/java/com/microsoft/azure/batch/protocol/models/MultiInstanceSettings.java

```diff
@@ -47,7 +50,7 @@
      *
      * @return the numberOfInstances value
      */
-    public int numberOfInstances() {
+    public Integer numberOfInstances() {
         return this.numberOfInstances;
```

Comment:
I know this is generated....but some numeric fields use type int (e.g. TaskExecutionInformation.java - retryCount) and some use type Integer - is this expected? Maybe it's a public vs private thing?

Reply:
Integer is the equivalent of "int?" in .NET

Reply:
Because this is optional parameter.